sunpy/io/_asdf.py

```diff
@@ -0,0 +1,113 @@
+from pathlib import Path
+from collections import OrderedDict
+
+import numpy as np
+
+import asdf
+
+from sunpy.io._header import FileHeader
+
+# extension_name = asdf.extension.ManifestExtension.from_uri(
+#     "asdf://astropy.org/astropy/manifests/units-1.0.0",converters = [UnitConverter()]
+# )
+# asdf.get_config().add_extension(extension_name)
```

Comment:
This is one of the solution which I tried by changing the tag name and adding it to asdf global config but I rejected it since it was having circular import error (its still there because I forgot to remove it )